docs/build-clio.md

```diff
@@ -19,6 +19,10 @@
 
 ### Conan Configuration
 
+> [!TIP]
+> To setup Conan automatically, you can run `.github/scripts/conan/init.sh`.
+> This will delete ~/.conan2 directory, if it exists, set up profiles and add Artifactory remote
```

Comment:
This makes me think that using default profile is perhaps not a very nice UX in some cases - if someone already uses conan to build some other projects for example

Reply:
Well, I'm saying explicitly what this does, so if you're already using Conan, you will know, that this script is not for you.

Script also supports $CONAN_HOME env dir, which allows to share/not share between projects. I will mention it.